stable/insights-agent/README.md

```diff
@@ -40,6 +40,25 @@ There are several different report types which can be enabled and configured:
 
 See below for configuration details.
 
+## Fleet Installation
+If you're installing the Insights Agent across a large fleet of clusters,
+it can be tedious to use the UI to create each cluster, then copy out the
+cluster's access token.
+
+To better serve customers with a large number of clusters, we've created a flow
+that allows you to easily deploy the Insights Agent across your fleet. You'll
+simply need to set the following flags:
+* `fleetInstall=true` - enable this flow
+* `insights.adminToken=xyz` - you can get this admin token from your organization's settings page at insights.fairwinds.com
+* `insights.tokenSecretName` - the name of the secret where Insights will store your cluster's token. We recommend `insights-token`
+* `insights.cluster` - the name you want to give this cluster in the Insights UI. You might want to auto-generate this from your kubectl context
+
+With these flags set, the Helm chart will create a new cluster in the Insights UI with the specified name
+(unless a cluster with that name already exists) before installing the agent.
+
+When reinstalling the agent, you can omit `adminToken` and `fleetInstall`, and simply specify `tokenSecretName`.
```

Comment:
reinstalling the agent to the same cluster right?

Reply:
:nod: